Automattic/jetpack
# Forms: Fix Multiple Choice response format

Fixes #30315

## Proposed changes:

* Fix Forms dashboard Multiple Choice response format

### Other information:


## Jetpack product discussion
N/A

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

Make sure you have the new Forms dashboard enabled

* Create a post and add a Form block
* Add a Multiple Choice field with a few options
* Publish the page, go to the frontend, select at least two options, and submit the form
* Go to the Forms dashboard and check the answers for the submitted response
* The multiple-choice options should now be separated by `,`
